README.md

```diff
@@ -7,7 +7,14 @@ This repository is used as a playground to run perception experiments. Each of t
 **Pre-requisite**: Before you run any of the commands below, make sure you are familiar with [Docker](https://www.docker.com/). If not, we highly recommend going through this [2-hour video](https://www.youtube.com/watch?v=fqMOX6JJhGo) which teaches you the basics of Docker.
 
 ## Getting Started
-To get started, run the `docker compose up` command, followed by the name of the service you are interested in (if you don't specify, you will end up launching every single service...). You can find all the services we have under [docker-compose.yml](./docker-compose.yml).
+To get started, run the following script to setup some environment variables
+
+```bash
+chmod +x initialize.sh
```

Comment:
Why would users have to set the execute bit? Can't you just set the execute bit one and commit it?

Reply:
oh I didn't know that. yeah chmod +x not needed